packages/syft/src/syft/core/smpc/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""SMPC AST Creation.
+
+__init__ file for SMPC. This defines various modules, classes and methods which we currently support.
+We create an AST for all these modules, classes and methods so that they can be called remotely.
+"""
+# stdlib
+from typing import Optional
+
+# relative
+from ...ast import add_classes
+from ...ast import add_methods
+from ...ast import add_modules
+from ...ast.globals import Globals
+from ..node.abstract.node import AbstractNodeClient
+from .store import CryptoStore
+
+from .protocol import *  # noqa: 401 isort: skip
+
+
+def create_smpc_ast(client: Optional[AbstractNodeClient] = None) -> Globals:
+    ast = Globals(client)
+
+    modules = ["syft", "syft.core", "syft.core.smpc", "syft.core.smpc.store"]
+
+    classes = [
+        (
+            "syft.core.smpc.store.CryptoStore",
+            "syft.core.smpc.store.CryptoStore",
+            CryptoStore,
+        ),
+    ]
+
+    methods = [
+        (
+            "syft.core.smpc.store.CryptoStore.get_primitives_from_store",
+            "syft.lib.python.List",
+        ),
+        ("syft.core.smpc.store.CryptoStore.store", "syft.lib.python.Dict"),
```

Comment:
I really like this approach.

Reply:
Do we have any restrictions on who is allowed to add values to anyone else's crypto store?

Reply:
Since we made an assumption of the Data Scientist to be the TTP, Since the Data Owners  coud connect to the DO, they could by default do it.But yeah we do not have restrictions, If DO was an account created for them they could do it.

Reply:
Yep. As @rasswanth-s said - only the DS will hold a pointer to the store and each party knows how to access their own store